test/clients/azure/client.go

```diff
@@ -22,9 +34,26 @@ func NewClientFromEnvironment() (*Client, error) {
 
 	subscriptionID := os.Getenv("AZURE_SUBSCRIPTION_ID")
 
+	var rpc externalapi.OpenShiftManagedClustersClient
+	focus := []byte(config.GinkgoConfig.FocusString)
+	switch {
+	case fakeRe.Match(focus):
+		fmt.Println("configuring the fake resource provider")
+		rpUrl := "http://" + fakerp.LocalHttpAddr
+		rpc = externalapi.NewOpenShiftManagedClustersClientWithBaseURI(rpUrl, subscriptionID)
```

Comment:
I think we need an additional focus here for [Admin] tests to use the the admin client.

Reply:
Its definitely accessible. You can have a look here: https://github.com/openshift/openshift-azure/blob/c2bbcdc4883eccca38d126b30bd72c5e683ca423/README.md